css/style.css

```diff
@@ -1,5 +1,105 @@
-/* General page styles */
-/* We should use a mobile first approach having out breack point at 768px */
+/* For mobile phones: */
+.button {
+  font-family: Roboto-Regular, sans-serif;
+  font-weight: normal;
+  padding-left: 20px;
+  padding-right: 20px;
+  max-width: 100%;
+  display: inline-block;
+  border-radius: 4px;
+  overflow: hidden;
+  color: white;
+  border: 0;
+}
+.button:focus {
+  outline: 0;
+}
+
+.button--sm {
+  min-width: 150px;
+  height: 30px;
+  font-size: 12px;
+}
+
+.button--md {
+  min-width: 200px;
+  height: 40px;
+  font-size: 16px;
+}
+
+.button--lg {
+  min-width: 320px;
+  height: 60px;
+  font-size: 22px;
+}
+
+.color--primary {
```

Comment:
All color classes for this component should be called `button--{color}`.

Reply:
does `max-width: 100%` actually works?

Reply:
Yes, this would stop the button from growing over the width of its container element.